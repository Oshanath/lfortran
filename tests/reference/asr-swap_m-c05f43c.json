{
    "basename": "asr-swap_m-c05f43c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../examples/swap_m.f90",
    "infile_hash": "57be4db42b2936988c6d21f742d23bb218d7cc2791b38e0e802c0ccd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-swap_m-c05f43c.stdout",
    "stdout_hash": "cf65e3fa50fc061781ac2ef58d3632098b6404459794a93013fba525",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}