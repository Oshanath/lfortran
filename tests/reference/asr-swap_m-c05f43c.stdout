(TranslationUnit (SymbolTable 1 {swap_m: (Module (SymbolTable 2 {swap_t: (Template (SymbolTable 3 {swap_: (Subroutine (SymbolTable 6 {tmp: (Variable 6 tmp Local () () Default (Derived 3 t []) Source Private Required .false.), x: (Variable 6 x InOut () () Default (Derived 3 t []) Source Private Required .false.), y: (Variable 6 y InOut () () Default (Derived 3 t []) Source Private Required .false.)}) swap_ [(Var 6 x) (Var 6 y)] [(= (Var 6 tmp) (Var 6 x) ()) (= (Var 6 x) (Var 6 y) ()) (= (Var 6 y) (Var 6 tmp) ())] Source Private Implementation () .false. .false.), t: (DerivedType (SymbolTable 5 {}) t [] Source Private ()), t_pair: (DerivedType (SymbolTable 4 {i: (Variable 4 i Local () () Default (Integer 4 []) Source Private Required .false.), x: (Variable 4 x Local () () Default (Real 4 []) Source Private Required .false.)}) t_pair [i x] Source Private ())}) [(TypeParameter t)] [] [])}) swap_m [] .false. .false.)}) [])
